frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -390,7 +418,11 @@ export default class PivotTable extends Component {
                     leftHeaderCellSizeAndPositionGetter
                   }
                   width={leftHeaderWidth}
-                  height={height - topHeaderHeight}
+                  height={
+                    needsScrollbarOffset()
+                      ? height - topHeaderHeight - 20
```

Comment:
Need to move this to a platform specific constant, but this value roughly works on Windows

Reply:
Maybe that could live in lib/browser since `isMac` is already there?

Also, this is a nitpick but I'd put the ternary inside the arithmetic expression to reduce duplication: `height - topHeaderHeight - (needsScrollbarOffset() ? SCROLLBAR_HEIGHT : 0)`

Reply:
Actually, maybe `needsScrollbarOffset` could be `scrollbarOffsetSize` instead which returns `SCROLLBAR_HEIGHT` or `0`